### DISCUSSION TITLE
Can LAN IPs access internet without v2ray client ?

### DISCUSSION THREAD

**Discussion by thinhnguyenhcm** [v2fly/v2ray-core#1790] [General]
Normally, I connect VPS has v2ray server from v2ray client.  But can I connect internet through my router has v2ray server without v2ray client.

![image](https://user-images.githubusercontent.com/41589802/168851143-880a428f-bb5a-41d1-8a1b-e39a865f8ea8.png)

**Comment by Cwek** [comment]
Do you understand v2ray and the logical relationship between client and server? 

Maybe you need to know about transparent proxy([dokodemo](https://www.v2fly.org/config/protocols/dokodemo.html#inboundconfigurationobject)), tun device, etc.
